.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,102 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Nightly Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 7 * * *' # run at 7 AM UTC, 2 AM ET.
+
+jobs:
+  nightly-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
If there's no name, does it put it ini the UI or not? I'd name this Checkout Branch.